### PR TITLE
fix(team_alerts): Remove Unassigned from Team Alerts Dropdown

### PR DESCRIPTION
**Pull request by mikellykels**
Update the color of the Unassigned user icon when it's selected. 

FIXES [WOR-762](https://getsentry.atlassian.net/browse/WOR-762)

### Before
_Unassigned Icon is difficult to see when selected_

Screen Shot 2021-04-12 at 11 14 42 PM

### After
Screen Shot 2021-04-13 at 11 27 12 PM

### CODE REVIEW

File: src/sentry/static/sentry/app/views/settings/incidentRules/ruleNameOwnerForm.tsx [hunk 1]
```diff
@@ -51,7 +51,6 @@ class RuleNameOwnerForm extends React.PureComponent<Props> {
                       model.setValue('owner', ownerValue);
                     }}
                     filteredTeamIds={userTeamIds}
-                    includeUnassigned
```
Comment (davidenwang) [inline src/sentry/static/sentry/app/views/settings/incidentRules/ruleNameOwnerForm.tsx:54]: I think these two forms are the only ones that use the `includeUnassigned` prop so if the end goal is to remove the option completely we should remove the `unassignedOption` in the `<SelectMembers>` component and related typing/props

**Review by davidenwang — COMMENTED** [review]
Looks good just one comment. 

I'm also wondering if we need to have some other way to make an alert unassigned since there is the possibility of having cross team alerts. cc @adhiraj

**Comment by mikellykels** [comment]
oh true true

**Comment by mikellykels** [comment]
changed this back to include the unassigned option and fixed the icon color when selected :)

File: src/sentry/static/sentry/app/components/selectMembers/index.tsx [hunk 2]
```diff
@@ -388,7 +389,15 @@ class SelectMembers extends React.Component<Props, State> {
         onInputChange={this.handleInputChange}
         onChange={this.handleChange}
         value={this.state.options?.find(({value}) => value === this.props.value)}
-        styles={styles}
+        styles={{
+          styles,
+          option: (provided, state: any) => ({
+            ...provided,
+            svg: {
+              color: state.isSelected ? 'white' : p => p.theme.gray400,
```
Comment (davidenwang) [inline src/sentry/static/sentry/app/components/selectMembers/index.tsx:397]: nit: can we use `p.theme.white` here?

**Review by davidenwang — APPROVED** [review]
nice looks real good 🌮

**Pull request merged**